CHANGELOG.md

```diff
@@ -15,31 +17,57 @@ rows in this table:
 SELECT table_rows FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'cromwell' AND table_name = 'JOB_STORE_SIMPLETON_ENTRY';
 ```
 
-### Delete Intermediate Outputs on PapiV2
+#### Execution Directory Layout (cache copies)
+
+Attempts to copy call cached results will no longer occupy the same directory as the first attempt to run the task, should the cache copy fail. 
```

Comment:
[ToL]
```suggestion
When an attempt to copy call cache results fails, its result will no longer occupy the same directory as the first attempt to run the task.
```

Reply:
My only nit with that rewording is that it's true even if the attempt to call cache does not fail.

Reply:
Ah, well if my version is straight up wrong then it's more than a nit

Reply:
I think the original commits the same sin. I'll try to make this more correct